proxyclient/m1n1/hv/__init__.py

```diff
@@ -1628,8 +1630,9 @@ def setup_adt(self):
                 pass
 
         if not self.smp:
+            boot_cpu_name = "cpu" + str(self.p.get_boot_cpu_idx())
             for cpu in list(self.adt["cpus"]):
-                if cpu.name != "cpu0":
+                if cpu.name != boot_cpu_name:
```

Comment:
simpler change without additional proxy commands would use the CPU's ADT node "state" property

Reply:
Yep.